src/index.js

```diff
@@ -62,15 +63,20 @@ history.listen((location) => {
 
 // Choose hash router for cdn only
 const Router = process.env.REACT_APP_ROUTER === 'hash' ? HashRouter : BrowserRouter;
-
-ReactDOM.render(
-  <AppServicesProvider forcedServices={forcedServices}>
-    <Router>
-      <App locale={locale} />
-    </Router>
-  </AppServicesProvider>,
-  document.getElementById('root'),
-);
+// If needed to turn app offline, set REACT_APP_OFFLINE true for the enviroment needed i.e. env.production
+const turn_app_offline = process.env.REACT_APP_OFFLINE === 'true';
```

Comment:
Isn't `process.env.REACT_APP_OFFLINE` a boolean value?

Reply:
It get's returned as string :/ we are using it like this in another places.